build_scripts/windows/azure-cli.wixproj

```diff
@@ -21,6 +12,15 @@
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' AND '$(MSBuildExtensionsPath32)' != '' ">$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' ">$(MSBuildExtensionsPath)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
   </PropertyGroup>
+  <!-- Local WiX -->
+  <PropertyGroup>
+    <LocalWixRoot>artifacts\wix</LocalWixRoot>
+    <WixToolPath>$(LocalWixRoot)</WixToolPath>
+    <WixTargetsPath Condition="Exists('$(WixToolPath)\Wix.targets')">$(WixToolPath)\Wix.targets</WixTargetsPath>
+    <WixTasksPath Condition="Exists('$(WixToolPath)\wixtasks.dll')">wixtasks.dll</WixTasksPath>
```

Comment:
I guess it's better to follow the convention of the official example and the previous code.

```suggestion
    <LocalWixRoot>artifacts\wix\</LocalWixRoot>
    <WixToolPath>$(LocalWixRoot)</WixToolPath>
    <WixTargetsPath Condition="Exists('$(WixToolPath)Wix.targets')">$(WixToolPath)Wix.targets</WixTargetsPath>
    <WixTasksPath Condition="Exists('$(WixToolPath)wixtasks.dll')">wixtasks.dll</WixTasksPath>
```

Reply:
Personally, I've never been a fan of potentially under-normalizing. Many paths - even MSBuild standard paths - may always be separator-terminated while others aren't. It's a cluster of different behaviors. There has almost never (and when it has been a problem, it's because of some third-party EXE invoked by a task) been a problem from including too many directory separators. Better safe than sorry, IMO.

Reply:
Thanks @heaths, then I guess the WiX document should be updated instead. 🤣